### PR TITLE
Memefixes 4

### LINKED ISSUE #203: [SUGGESTION] | Sisterhood of Elune background is too bright
**Describe your suggestion in full detail below:**
Srsly, this brightness hurts my eyes.

<details>
<summary>Click to expand</summary>

ck2_5

</details>

### LINKED ISSUE #1174: [SUGGESTION] | Add rescue frends to intrigue focus
I think it would be very good to add the ability to rescue your friends from prison fot intrigue focus. 

It is a very stupid decision of paradox to make it possible to save only family

### PR DESCRIPTION
**Pull request by MemeWizard**
#### Changelog:
- Removed vanilla religions from the ladder.
- Allowed to rescue friends and male consorts from prison through intrigue focus. Closes #1174.
- Your character no longer can sacrifice characters that can't be sacrificed in primitive rites.

#### Dev changelog:
- Changed the Sisterhood of Elune textures. Fixed #203.
- Added more info about what level of the whispers your character will get to "write down whispers" decision effects.
- Rebel emperors no longer can counts as emperors in Hakkari sacrificial calculations and no longer give a character more sacrificial points.
- Fixed bug where your character couldn't join a mod secret society through "cultist recruiting your character" event if you don't have the Monks and Mystics because recruiting event is M&M-locked. Now if you don't have M&M, your character will join society directly, without recruiters.
- Fixed that `b_brent` didn't have a unique forsaken cultural name.

#### Code changelog (ignore if you can't read CK2 script):
- Added effect for giving the Doomhammer to a character.
- Deleted obsolete sacrifice effects
- Fixed wc_treasury.gfx errors by deleting extra brackets
- Doubled burned_hearts_points where I forgot to do so in #1210

### CODE REVIEW

File: common/scripted_effects/wc_scripted_effects.txt [hunk 1]
```diff
@@ -708,73 +708,6 @@ drain_void_source_effect = {
 		}
 	}
 }
-sacrifice_all_prisoners_effect = {
```
Comment (zumbak04) [inline common/scripted_effects/wc_scripted_effects.txt]: Still used in the files. You can't access the event though.

File: common/scripted_effects/wc_scripted_effects.txt [hunk 2]
```diff
@@ -708,73 +708,6 @@ drain_void_source_effect = {
...
-				
-				prisoner = yes
-				can_be_sacrificed_trigger = yes
-				host = { character = ROOT }
-			}
-			prev_sacrifice_this_effect = yes
-		}
-	}
-}
-prev_sacrifice_this_effect = {
```
Comment (zumbak04) [inline common/scripted_effects/wc_scripted_effects.txt]: Still used in the files.

File: common/scripted_effects/wc_scripted_effects.txt (hunk 1 above)
Comment (MemeWizard) [reply common/scripted_effects/wc_scripted_effects.txt]: Removed events

File: common/scripted_effects/wc_scripted_effects.txt (hunk 2 above)
Comment (MemeWizard) [reply common/scripted_effects/wc_scripted_effects.txt]: reworked

File: interface/wc_decision_icons.gfx [hunk 3]
```diff
@@ -156,10 +156,6 @@ spriteTypes = {
 		name = "GFX_corrupt_the_sunwell"
 		texturefile = "gfx\\interface\\decision_icons\\restore_the_sunwell.dds"
 	}
-	spriteType = {
-		name = "GFX_blood_rituals"
-		texturefile = "gfx\\interface\\decision_icons\\blood_rituals.dds"
```
[the remark this answers is not in the document]
Comment (MemeWizard) [reply interface/wc_decision_icons.gfx]: Moved to artist resources

File: decisions/way_of_life_decisions.txt [hunk 4]
```diff
@@ -612,28 +612,36 @@ targetted_decisions = {
...
-						}
+				OR = {
+					liege = { character = FROM }
+					is_close_relative = FROM
+					is_friend = FROM
+				}
+			}
+			trigger_else = {
+				is_female = yes
+				is_consort = yes
```
[the remark this answers is not in the document]
Comment (MemeWizard) [reply decisions/way_of_life_decisions.txt]: Allowed to rescue males too

**Comment by MemeWizard** [comment]
<3

**Pull request merged**